grammars/php.cson

```diff
@@ -487,233 +480,506 @@
         ]
       }
       {
-        'begin': '(?=<<<\\s*(\'?)([a-zA-Z_]+[a-zA-Z0-9_]*)(\\1)\\s*$)'
+        'begin': '(?=<<<\\s*\'([a-zA-Z_]+[a-zA-Z0-9_]*)\'[ \\t]*$)'
         'end': '(?!\\G)'
         'name': 'string.unquoted.heredoc.nowdoc.php'
```

Comment:
that's what I was thinking but I didn't know if there was a reason for the `.heredoc` (styling or whatever)

I can change it if you want